requirements.txt

```diff
@@ -11,5 +11,6 @@ mistralai==0.0.8
 python-dotenv==1.0.0
 pre-commit==3.6.0
 lamini==2.0.4
+tabulate==0.9.0
```

Comment:
What do you need this for?

Reply:
It's nice, but I would avoid that. It's a package that is used in other applications. It's not a standalone solution :)